package Session15;

public class TryCatch_3 {

	public static void main(String[] args)
	{
//		TryCatch_3 tc= new TryCatch_3();
//		tc.division(9, 3);// if we take (9,0) its show error arithmetic exception
//	}
//public void division(int Ny, int Dy)
//{
//	int division = Ny/Dy;
//	System.out.println("The division of input --->"  +Ny+ "and" +Dy+ "is --->" + division);
//}

//try catch

//TryCatch_3 tc= new TryCatch_3();
//try
//{
//	tc.division1(9, 0);
//	System.out.println();
//}
//catch(Exception e)
//{
//	e.printStackTrace();
//	
//}
//public void division1(int Ny1,int Dy1)
//{
//int division1 = Ny1/Dy1;
//
//System.out.println("The division of input --->"  +Ny1+ "and" +Dy1+ "is --->" + division1);
//}
}
	
	
	
}
